.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "AWS CDK & Python Development Environment",
+  "image": "mcr.microsoft.com/devcontainers/base:ubuntu-22.04",
```

Comment:
```suggestion
"postCreateCommand": "pip install --user -U pre-commit && pre-commit install-hooks",
```

Reply:
You are overwriting the existing `postCreateCommand` defined at the bottom of the file.